### ISSUE TITLE
Don't remove Wrap Instruction after match

### ISSUE THREAD

**Issue by fabianmeyer** [renatoathaydes/osgi-run#53]
When the manifest of a "wrapInstruction" matches a file pattern, the instruction is removed from the config:

BndWrapper.groovy
```
  Map<String, Object[]> config = getWrapConfig( wrapInstructions, jarFile )
        def consumeValue = { String key ->
            Object[] items = config.remove( key )
            if ( items ) items.join( ',' )
            else null
        }
```

Hence, the instruction is not applied to subsequent jar files, matching the pattern. E.g. 

```
runOsgi {
  bundles += subprojects

  wrapInstructions {
      manifest( ~/.*/ ) {
          instruction 'Import-Package', '!org.scalatest*', '!org.scalacheck*', '!org.specs*','*'
      }
  }
}
```

is only applied to the first match.

**Comment by renatoathaydes** [comment]
Fixed in 1.5.2.

**Comment by fabianmeyer** [comment]
Sorry, I was quite busy in the last days. Thanks for fixing it!
